redbot/cogs/cleanup/cleanup.py

```diff
@@ -115,13 +121,13 @@ def message_filter(message):
         return collected
 
     @commands.group()
-    @checks.mod_or_permissions(manage_messages=True)
```

Comment:
Could you merge current `V3/develop` into this branch so that these differences don't show up in this diff?